ext/js/language/ja/japanese.js

```diff
@@ -730,7 +730,7 @@ export function distributeFuriganaInflected(term, reading, source) {
 // Miscellaneous
 
 /**
- * @param {number} codePoint
+ * @param {number | undefined} codePoint
```

Comment:
Why is 'undefined' here and not handled outside of the function?

Reply:
This is for not getting the typing warning as codePointAt() can return undefined. 
Just check other usages in the codebase, looks like adding `/** @type {number} */` is the correct way to handle this.